BlockchainSdkTests/Solana/SolanaEd25519Slip0010Tests.swift

```diff
@@ -91,14 +55,21 @@ final class SolanaEd25519Slip0010Tests: XCTestCase {
             contractAddress: nil
         )
 
-        let expected = expectation(description: "Waiting for response")
+        do {
+            try await manager.send(transaction, signer: coinSigner).async()
+        } catch let sendTxError as SendTxError {
+            guard let castedError = sendTxError.error as? SolanaError else {
+                XCTFail("Wrong error returned from manager")
```

Comment:
как вариант можно
```suggestion
               throw error
```
Это выкинет ошибку выше, т.е. в сам тест )